raco/myrial/parser.py

```diff
@@ -470,6 +477,12 @@ def p_statement_assign(p):
         p[0] = ('ASSIGN', p[1], p[3])
 
     @staticmethod
+    def p_statement_idbassign(p):
+        'statement : unreserved_id COLON EQUALS LBRACKET emit_arg_list \
```

Comment:
Curious why we're introducing a new token `:=` for assignment in this context. Is there some ambiguity if we used `=`? I doubt it since the parse of the `IDBASSIGN` production calls exactly the same code as an ordinary assignment statement. In any case, I find `:=` a little confusing since it's so similar to the Datalog `:-` symbol that we're expressing as `<-`.